src/components/SecondaryPanes/Frames/index.js

```diff
@@ -176,6 +182,7 @@ class Frames extends Component {
     return dom.div(
       { className: "pane frames" },
       this.renderFrames(frames),
+      WhyPaused(),
```

Comment:
Absolutely agree. Had a thought about making it a stateless component, but didn't know what do you guys think about that.